src/components/item-card-content.js

```diff
@@ -1,54 +1,54 @@
 import React from 'react'
-import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 import DisplaySelector from './display-selector'
 import { ItemTypes } from '@kleros/gtcr-encoder'
 import { Button } from 'antd'
-import { Link } from 'react-router-dom'
+import useNavigateAndScrollTop from 'hooks/navigate-and-scroll-top'
+import { StyledItemCol } from './light-tcr-card-content'
 
-const StyledItemCol = styled.div`
-  margin-bottom: 8px;
-  text-align: center;
-`
+const ItemCardContent = ({ item, chainId, tcrAddress }) => {
+  const navigateAndScrollTop = useNavigateAndScrollTop()
 
-const StyledButton = styled(Button)`
-  pointer-events: auto;
-  text-transform: capitalize;
-`
-
-const ItemCardContent = ({ item, chainId, tcrAddress }) => (
-  <div
-    style={{
-      height: '100%',
-      display: 'flex',
-      flexDirection: 'column',
-      justifyContent: 'space-between',
-      alignItems: 'center'
-    }}
-  >
-    <div>
-      {item.columns
-        .filter(
-          col =>
-            col.isIdentifier ||
-            col.type === ItemTypes.IMAGE ||
-            col.type === ItemTypes.FILE
-        )
-        .map((column, j) => (
-          <StyledItemCol key={j}>
-            <DisplaySelector
-              type={column.type}
-              value={column.value}
-              allowedFileTypes={column.allowedFileTypes}
-            />
-          </StyledItemCol>
-        ))}
+  return (
+    <div
+      style={{
+        height: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+        alignItems: 'center'
+      }}
+    >
```

Comment:
is there a particular reason to use in-line styling here instead of styled components?

Reply:
no reason, that was there before, but I will refactor it as styled component